tests/functional/ext/private_import/private_import.txt

```diff
@@ -0,0 +1,11 @@
+import-private-name:5:0:5:24::Imported private module (_world):UNDEFINED
+import-private-name:6:0:6:25::Imported private module (_world):UNDEFINED
+import-private-name:7:0:7:23::Imported private object (_house):UNDEFINED
+import-private-name:8:0:8:29::Imported private object (_b, _d):UNDEFINED
+import-private-name:9:0:9:30::Imported private module (_city):UNDEFINED
+import-private-name:10:0:10:28::Imported private object (_d):UNDEFINED
+import-private-name:12:0:12:13::Imported private module (_house):UNDEFINED
+import-private-name:13:0:13:29::Imported private module (_house, _chair, _stair):UNDEFINED
```

Comment:
Can we make it so that when there are multiple imports we say: `Imported private moduleS`? Obviously without the capitalisation.